tests/foreman/cli/test_host.py

```diff
@@ -1637,6 +1638,41 @@ def test_positive_apply_errata(self):
 
     @tier3
     @run_only_on('sat')
+    def test_positive_apply_security_erratum(self):
+        """Apply security erratum to a host
+
+        :id: 4d1095c8-d354-42ac-af44-adf6dbb46deb
+
+        :Assert: erratum is recognized by the
```

Comment:
@oshtaier we missed this `:Assert:` which has been renamed to `:expectedresults:`